babble/nlp/engine.py

```diff
@@ -228,6 +230,14 @@ def _evaluate_classifier(
                 return slot, phrase
         return None, phrase
 
+    def _filter_intents_by_lenght(self, phrase: str) -> List[Dict]:
+        phrase_len = len(phrase.split(" "))
+        min_lim, max_lim = (phrase_len - 3, phrase_len + 3)
+        intents_to_test = [
+            intent for intent in self.intents if min_lim < intent["len_rule"] < max_lim
+        ]
+        return intents_to_test
+
```

Comment:
Idea for later: Do you think it is possible to "connetect" these limits with the limits from validity calculation? I think they are quite similar.

Reply:
we can think about it later. I am not sure that the validity calculation/ filtering approach would not change.